### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mocsharp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,24 @@
 💡 If you want to know more about MONAI Deploy WG vision, overall structure, and guidelines, please read [MONAI Deploy](https://github.com/Project-MONAI/monai-deploy) first.
 
 # MONAI Deploy Messaging
-Messaging layer for MONAI Deploy clinical data pipelines system.
+
+The MONAI Deploy Messaging library for MONAI Deploy clinical data pipelines system enables users to extend the system to external message broker services by implementing the [IMessageBrokerPublisherService](src/Messaging/API/IMessageBrokerPublisherService.cs) and [IMessageBrokerSubscriberService](src/Messaging/API/IMessageBrokerSubscriberService.cs) APIs.  The APIs allow users to plug in any other message broker services, such as [Apache Kafka](https://kafka.apache.org/intro) and [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
```
Comment (dbericat) [inline README.md:9]: Same double space '.  The API'

File: README.md [hunk 2]
```diff
@@ -5,15 +5,24 @@
...
+
+- [RabbitMQ](https://www.rabbitmq.com/)*
+
+\* Services provided may not be free or requires special license agreements.  Please refer to the service providers' website for additional terms and conditions.
+
+If you would like to use a message broker service not listed above, please file an [issue](https://github.com/Project-MONAI/monai-deploy-messaging/issues) and contribute to the repository.
 
 ## Contributing
 
 For guidance on making a contribution to MONAI Deploy Workflow Manager, see the [contributing guidelines](https://github.com/Project-MONAI/monai-deploy/blob/main/CONTRIBUTING.md).
```
Comment (dbericat) [inline README.md:21]: Change Workflow Manager to Messaging

File: README.md [hunk 3]
```diff
@@ -5,15 +5,24 @@
...
 
 Join the conversation on Twitter [@ProjectMONAI](https://twitter.com/ProjectMONAI) or join our [Slack channel](https://forms.gle/QTxJq3hFictp31UM9).
 
-Ask and answer questions over on [MONAI Deploy Workflow Manager's GitHub Discussions tab](https://github.com/Project-MONAI/monai-deploy-workflow-manager/discussions).
+Ask and answer questions over on [MONAI Deploy's GitHub Discussions tab](https://github.com/Project-MONAI/monai-deploy/discussions).
```
Comment (dbericat) [inline README.md:25]: Same comment. Discussions at the messaging repo level are better than in the parent?

File: README.md [hunk 4]
```diff
@@ -5,15 +5,24 @@
...
 
 ## Links
```
Comment (dbericat) [inline README.md:28]: Missing links?

**Review by dbericat — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (mocsharp) [reply README.md:28]: There are links.

File: README.md (hunk 3 above)
Comment (mocsharp) [reply README.md]: Updated to https://github.com/Project-MONAI/monai-deploy-messaging/discussions

File: README.md (hunk 2 above)
Comment (mocsharp) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (mocsharp) [reply README.md]: Fixed

**Review by dbericat — APPROVED** [review]
LGTM

**Pull request merged**